js/src/redux/providers/signerMiddleware.js

```diff
@@ -120,7 +120,7 @@ export default class SignerMiddleware {
         });
     }
 
-    handlePromise(this._api.signer.confirmRequest(id, { gas, gasPrice }, password));
+    handlePromise(this._api.signer.confirmRequest(id, { gas, gasPrice }, password.password));
```

Comment:
Is this really working ? Would at least have `typeof password === 'string' ? password : password.password` or equivalent because it looks strange.